content/docs/lifting-state-up.md

```diff
@@ -110,15 +110,15 @@ class Calculator extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/jGBryx?editors=0010)
+[**Cobalah di CodePen**](https://codepen.io/gaearon/pen/jGBryx?editors=0010)
 
-We have two inputs now, but when you enter the temperature in one of them, the other doesn't update. This contradicts our requirement: we want to keep them in sync.
+Kami memiliki dua input sekarang, tetapi ketika anda memasukkan suhu di salah satu dari mereka, yang lain tidak memperbarui. Ini bertentangan dengan persyaratan kami: kami ingin tetap menyinkronkannya.
```

Comment:
`We have two inputs` sepertinya konteksnya `Kita sekarang telah memiliki dua input`

`This contradicts our requirement: we want to keep them in sync.`

sepertinya lebih cocok:
`Hal ini bertentangan dengan persyaratan kita: kita ingin agar mereka tetap sinkron`